src/components/Quiz/useQuizStore.ts

```
@@ -172,67 +172,72 @@
             });
-            // Add to user answers as skipped
-            addUserAnswer(currentQuestion.id);
           }
         },
 
         addUserAnswer: (questionId, answerId) => {
-          const { userAnswers, bySimulado, byAtividade, byAula } = get();
+          const { bySimulado, byAtividade, byAula } = get();
           const quiz = bySimulado || byAtividade || byAula;
-          const question = quiz?.questions.find((q) => q.id === questionId);
+          
```

ESLint

[error] Delete `··········`

(prettier/prettier)